demaceo/whats-cookin
# Feature/user section

### What's this PR do?
**Allows a `currentUser` to:**
- Favorite & unfavorite a recipe in the Recipes To Cook section.
- Search by ingredient & recipe & return the results.

### How should this be manually tested?
1. `open index.html` in Terminal.
2. In the browser, choose a `currentUser` from the **Accounts dropdown list**.
3. Type a recipe or ingredient in the search bar & view results.
4. To add a recipe to `currentUser.recipesToCook`, click the **bookmark-icon** that is located next to the desired recipe.
5. To see that it has been saved, go to the **User dropdown menu**, & click **Recipes To Cook**.
